### ISSUE TITLE
gB Open QAT ADM2

### ISSUE THREAD

**Issue by jingjli1** [wmgeolab/geoBoundaries#3000]
Names are number codes.
Correct meta and resubmit.

**Comment by graciegundlach** [comment]
Running into a lot of difficulty with this project, so I am resubmitting the zones I found names for but hope to pass this project onto someone more experienced! 
1) Names for the "Zones" (their ADM 3 level) are very long, longer than what I can type into the attribute field on ARCGIS. For example, zone #98 is "Al Wakra - Khor Al Udaid, Nega Al Fagara, Aamra, Negyan Al Sada, Wadi Abal Gherban, Legsaira, Eglat Zuwayyed, Lekhfoos". I am using this source: https://www.psa.gov.qa/en/statistics/Statistical%20Releases/General/QMS/QMS_PSA_75_April_2020.pdf
2) Some of the zones have the same name. for example zones 22 & 23 are both " Bin Mahmoud"
3) Our map does not line up with the map from the Qatari government linked above which is from 2020. Not sure which one is "correct" or most up to date
4) Several missing zones in our map, we have 79 of the supposed 98 zones total. Several of the zone names are left empty on this source https://www.iloveqatar.net/guide/living/zones-of-qatar-zoning-regulations-mme-albaladiya

**Comment by ejott** [comment]
The names Gracie found for this seem to be names of the districts, which is probably why there are so many names for each. I guess zones with the same "name" split the districts up. According to the 2020 Census, there are 91 zones, and their official names are simply numbers, not the district names. I could not find any evidence of other official district names as of 2020. The Open Data Portal where the boundaries available right now were from (no longer available) has the names Gracie found, but there are only 90 zones and those are from April 2010. Qatar’s GIS department (CGIS) has the zones available to view/download (https://services.gisqatar.org.qa/server/rest/services/Vector/Zones/MapServer), but their website lacks license information. 

TL;DR: The ADM2 zones don’t have official non-number names, and I can’t find any sources for ADM2 boundaries that have the correct license.
